src/bash/Makefile

```diff
@@ -10,7 +10,11 @@ $(addprefix $(DEST)/, $(MAIN_TARGET)): $(DEST)/% :
 	dget -u https://launchpad.net/debian/+archive/primary/+sourcefiles/bash/$(BASH_VERSION_FULL)/bash_$(BASH_VERSION_FULL).dsc
 
 	pushd bash-$(BASH_VERSION_MAJOR)
+ifeq ($(CROSS_BUILD_ENVIRON), y)
+	CONFIG_SITE=/etc/dpkg-cross/cross-config.$(CONFIGURED_ARCH) LD_LIBRARY_PATH=$(CROSS_LIB_PATH) dpkg-buildpackage -us -uc -b -a$(CONFIGURED_ARCH) -Pcross,nocheck -j$(SONIC_CONFIG_MAKE_JOBS) --admindir $(SONIC_DPKG_ADMINDIR)
```

Comment:
>CONFIG_SITE=/etc/dpkg-cross/cross-config.$(CONFIGURED_ARCH) LD_LIBRARY_PATH=$(CROSS_LIB_PATH) [](http://example.com/codeflow?start=1&length=93)

Could you generalize the environment and extract it into slave.mk?
This could also benefit the original `dpkg-buildpackage` command line below.

Reply:
CONFIG_SITE appears in a lot of Makefiles. 
Let us first agree on what I need to do, so that I do it once and for sure.

You want me to create a new environment variable, say, **CROSS_DPKG_FLAGS**=**CONFIG_SITE=/etc/dpkg-cross/cross-config.$(CONFIGURED_ARCH) LD_LIBRARY_PATH=$(CROSS_LIB_PATH)** in slave.mk and use only it everywhere where the both flags are currently used.
Is it what you mean ?

Reply:
Hi @qiluo-msft . I have already fixed all issues you had requested to fix for which I had no questions. The only remaining issues are those for which I asked you for clarifications and advise.
Please answer my questions to allow me to proceed.
Thanks.

Reply:
My concern is that you changed so many Makefile files in the similar fashion. Could you change it only in one place (for example in slave.mk), and it will be automatically applied to all the lower level Makefile.

Reply:
Seem like you need to change Makefile in many files. Could you unified if/else into one line of dpkg-buildpackage.